package.json

```diff
@@ -135,7 +137,7 @@
     "test": "jest",
     "test:debug": "node --inspect node_modules/.bin/jest --watch --runInBand",
     "view-cover": "npm run cover -- --report=html && opn ./coverage/index.html",
-    "package": "npm-run-all clean lint test build build:prod",
+    "package": "npm-run-all reinstall-react clean lint build:prod clear-react",
```

Comment:
When we package react-digraph, we don't actually need the full build. I removed the build script from this and only included build-prod to help reduce the package size.